docs/cas-server-documentation/logging/Logging-SysLog.md

```diff
@@ -29,6 +29,8 @@ messages needs to be routed over to this instance:
 
 ```
 
+> :warning: Mapped Diagnostic Context (MDC) may contain the password. Setting includeMDC to true sends clear password to syslog.
```

Comment:
Please use the appropriate `alert alert-warning` construct to show warnings.

```
<div class="alert alert-warning">
  <strong>WATCH OUT!</strong><br />xyz
</div>
```

Also make sure reference to ode/config elements such as includeMDC and `true` are formatted correctly via code tags.